robottelo/ui/role.py

```diff
@@ -85,3 +85,29 @@ def add_permission(self, role_name, resource_type=None,
             self.click(tab_locators['roles.tab_location'])
             self.configure_entity(location, FILTER['filter_loc'])
         self.click(common_locators['submit'])
+
+    def get_resources(self, role_name):
```

Comment:
`get_permissions` can be a good method easily if it starts using builtin search, but right now it doesn't use it so it still requires update. Or am i missing something?